src/basics.md

```diff
@@ -326,13 +327,185 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-run-piped-external-commands]: #ex-run-piped-external-commands
+<a name="ex-run-piped-external-commands"></a>
+## Run piped external commands
+
+[![clap-badge]][clap] [![cat-os-badge]][cat-os]
+
+Shows the `<N>`th biggest files/subdirectories in the specified
+`<directory>`. It spawns three Unix processes. It is equivalant to
+run: `du -ah <directory> | sort -hr | head -n <N>`.
+
+```rust,no_run
+# extern crate clap;
+
+# use clap::{App, Arg, ArgMatches};
+use std::error::Error;
+use std::fs::metadata;
+use std::process::{exit, Command, Stdio};
+
+static DU: &'static str = "du";
```

Comment:
I'd remove these constants. The example will be shorter and more readable if we write the tool name and argument list directly in the `Command::new` builder chain